src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -871,6 +871,10 @@ public AsyncIndex getAsyncIndexConnection(String indexName) throws PineconeValid
         return new AsyncIndex(connection, indexName);
     }
 
+    public Inference getInference() {
```

Comment:
This feels a little odd to me as a getter. Wondering if just `pc.inference().embed(...)` is better?

Reply:
Yes, I like pc.inference().embed(...) more

Reply:
I think [pc.inference().embed(...) definitely feels more like python usage and it aligns with rest of our SDKs but given its java, do you feel like users will directly call pc.inference().embed()? And pc.inference() will still return the Inference object regardless, so the naming does make sense if not compared with other SDKs.

Reply:
Also for dataplane opeartions, we have getIndexConnection() and getAsyncIndexConnection().